packages/ui/src/lib/data.ts

```diff
@@ -1,6 +1,13 @@
+const getChainEnvPrefix = () => {
+  const chainEnv = process.env.NEXT_PUBLIC_CHAIN_ENV;
+  return chainEnv ? `${chainEnv}.` : "";
+};
+
+const createTorusUrl = (subdomain: string) =>
+  `https://${getChainEnvPrefix()}${subdomain}.torus.network`;
```

Comment:
Wait, is this the right order? e.g.

[dao.testnet.torus.network](https://dao.testnet.torus.network/)